cnf-certification-test/manageability/suite.go

```diff
@@ -57,3 +65,29 @@ func testContainersImageTag(env *provider.TestEnvironment) {
 		ginkgo.Fail("Containers have been found that are missing tag(s) on image.")
 	}
 }
+
+// The name field in the ContainerPort section must be of the form <protocol>[-<suffix>] where <protocol> is one of the following,
+// and the optional <suffix> can be chosen by the application. Allowed protocol names: grpc, grpc-web, http, http2, tcp, udp.
+var allowedProtocolNames = map[string]bool{"grpc": true, "http": true, "http2": true, "tcp": true, "udp": true}
```

Comment:
I don't believe this is an exhaustive list, maybe we could scrape the following db for the ip protocol keyword:
https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml

Reply:
The problem I see is that the requirement is too vague. In the list you propose I haven't found "grpc" or "http2", which are given as valid examples of protocol names. I think this requirement needs some clarification.

Reply:
yes, I agree. fyi, I found this project that already has a json with the port mappings: https://github.com/silverwind/port-numbers/blob/master/ports.json

Reply:
As discussed internally, let's keep the current implementation for now.